torch/_inductor/config.py

```diff
@@ -961,6 +965,14 @@ class triton:
         os.environ.get("TORCHINDUCTOR_PERSISTENT_REDUCTIONS", "1") == "1"
     )
 
+    # For small output size reductions uses cross thread-block synchronization to gain more parallelism
+    cooperative_reductions = (
+        os.environ.get("TORCHINDUCTOR_COOPERATIVE_REDUCTIONS", "0") == "1"
```

Comment:
Are you planning on keeping these off by default?

Reply:
I plan to turn it on once I tune the heuristics.